### DISCUSSION TITLE
Printer Killed during homing

### DISCUSSION THREAD

**Discussion by ni1zzz** [mriscoc/Ender3V2S1#1222] [Q&A] [answered]
I loaded the firmware and tried homing all axis. No axis move and after some seconds the display says: printer killed please turn off. 
Then i turned the printer off and back on again. Tried homing and the same problem occurred. 
I tried this several times.

**Comment by mriscoc** [comment, marked as answer]
Perhaps a incorrect board version firmware.

**Reply by KrBarrow07** [reply]
I am having the same problem

**Comment by KrBarrow07** [comment]
Ni1zzz, did you ever get this resolved as I am having the same problem?
